src/Compilers/Core/Portable/Compilation/SemanticModel.cs

```diff
@@ -71,7 +72,12 @@ public SyntaxTree SyntaxTree
         /// <returns></returns>
         public IOperation GetOperation(SyntaxNode node, CancellationToken cancellationToken = default(CancellationToken))
         {
-            return this.GetOperationCore(node, cancellationToken);
+            // TODO: https://github.com/dotnet/roslyn/issues/9270
+            if (this.Compilation.IsIOperationFeatureEnabled())
+            {
+                return GetOperationCore(node, cancellationToken);
+            }
+            throw new InvalidOperationException(CodeAnalysisResources.IOperationFeatureDisabled);
```

Comment:
Inspired partly by this and partly by the regrettable reality that we can't guarantee that we can't guarantee that getting operations from syntax nodes won't throw exceptions, the code in the analyzer driver/executor that gets operations should be wrapped in a try/catch.